import_data/tasks.py

```diff
@@ -15,18 +19,35 @@
 from pydantic import BaseModel
 from pydantic.datetime_parse import parse_datetime
 
-
+psycopg2.extensions.register_adapter(dict, psycopg2.extras.Json)
```

Comment:
Out of curiosity, is it necessary to use both `register_adapter` and `register_hstore` here ?

Reply:
Not a single one, this one is actually only intended for Postgres' JSON fields.